### ISSUE TITLE
Add info for PyPI logo usage in a very visible spot

### ISSUE THREAD

**Issue by leahecole** [pypa/warehouse#9744]
Hi! The PSF website has [clear guidance for their logo usage](https://www.python.org/community/logos/) which makes it easy for Pythonistas like me to appropriately feature the logo in presentations and elsewhere when appropriate. I have looked around and it seems that a similar page does not exist for the PyPI logo - it would be great to have a page with resource (logo) downloads, guidance on logo usage and branding things, so that folks can appropriately represent PyPI when needed. Thank you!

**Comment by di** [comment]
Thanks, I think that would be relatively easy to add to https://pypi.org/help/

**Comment by ewjoachim** [comment]
I think if we take the time to state here what info we need to documents (i.e. what are the actual rules regarding this logo), this could be a Good First Issue?

**Comment by leahecole** [comment]
I agree - the part that's difficult is not knowing what the rules are 😄

**Comment by di** [comment]
We now have `pypi.org/trademarks`, added in https://github.com/pypi/warehouse/pull/12405. We should add this to https://pypi.org/help/ as well.
